### PR TITLE
Feature/asp 122 data in trace info

### PR DESCRIPTION
**Pull request by ramavath**
TraceInfo is added for following functionality

Arrears of Pay

Holiday taken but not Paid

Notice worked but not paid

### CODE REVIEW

**Review by bpowner-nomensa — APPROVED** [review]
Looks good

File: Insolvency.CalculationsEngine.Redundancy.BL/Serializer/Extensions/TraceInfoSerializer.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+{
+    public static class TraceInfoSerializer
+    {
+        private static JsonSerializerSettings jsonSettings { get; set; } = new JsonSerializerSettings
+        {
+            DateFormatString = "dd/MM/yyyy hh:mm:ss",
+            Formatting = Formatting.None
+            
+        };
+        public static string ConvertToJson()
```
Comment (MatthewWilkes) [inline Insolvency.CalculationsEngine.Redundancy.BL/Serializer/Extensions/TraceInfoSerializer.cs:20]: This is a better name!

File: Insolvency.CalculationsEngine.Redundancy.BL/Services/Implementations/ArrearsOfPayCalculationsService.cs [hunk 2]
```diff
@@ -71,7 +72,7 @@ public class ArrearsOfPayCalculationsService : IArrearsOfPayCalculationsService
 
                     //is this day a working day?
                     if (await date.IsEmploymentDay(data.ShiftPattern))
-                    {
```
Comment (MatthewWilkes) [inline Insolvency.CalculationsEngine.Redundancy.BL/Services/Implementations/ArrearsOfPayCalculationsService.cs:74]: Please avoid reformatting files as part of functionality commits in future. It makes the diffs bigger than they need to be.

**Review by MatthewWilkes — APPROVED** [review]
LGTM!

**Pull request merged**